src/Mod/Draft/draftguitools/gui_beziers.py

```diff
@@ -60,14 +60,14 @@ def GetResources(self):
         """Set icon, menu and tooltip."""
 
         return {'Pixmap': 'Draft_BezCurve',
-                'Accel': "B, Z",
-                'MenuText': QT_TRANSLATE_NOOP("Draft_BezCurve", "Bezier curve"),
-                'ToolTip': QT_TRANSLATE_NOOP("Draft_BezCurve", "Creates an N-degree Bezier curve. The more points you pick, the higher the degree.\nCTRL to snap, SHIFT to constrain.")}
+                'Accel': 'B, Z',
+                'MenuText': QT_TRANSLATE_NOOP('Draft_BezCurve', 'Bézier curve'),
```

Comment:
Why are the `"` changed to `'`? Standard should be " as in the rest of the file and PEP8

Reply:
@Roy-043 this appears to have happened at a few places in this changeset -- can you revise to use double quotes?

Reply:
If you look carefully you will see that in the original file a mix on single and double quotes are used while defining the same dictionary. So there is no 'standard' in the file. Also: the original file was recently refactored.

But I'll fix this anyway.

Reply:
Hmm, actually there was no need to fix this:
https://forum.freecadweb.org/viewtopic.php?f=8&t=65753

@benj5378 Can you confirm please.

Reply:
I don't see any specific conclusion in the forum thread? They mention some personal exceptions for when they wouldn't use PEP8.

Regarding Zolko's comment, I agree with him, but it's for a whole other case: naming of variables and functions. Not code formatting regarding use of symbols, spaces, line breaks etc.

I noticed that the file is mixed, but forward on, I'd personally prefer it was always changed towards PEP8.

Conclusion, do whatever you want and I'll make a seperate PR afterwards doing PEP8

Reply:
PEP8 allows both single and double quotes.
Link provided in 1st forum topic answer:
https://www.python.org/dev/peps/pep-0008/#string-quotes

So your initial comment here does not make sense.

> Standard should be " as in the rest of the file

> I noticed that the file is mixed

And with that you initial comment makes even less sense.

Striving for consistent use of double or single quotes in the Python code of FreeCAD is a lost cause if ever I saw one. But, enough said, let's merge!

Reply:
As you say 😎